config/dispatcher/dispatcher.ex

```diff
@@ -153,6 +161,30 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://resource/gender-codes/"
   end
 
+  match "/minister-positions/*path", %{ accept: [:json], layer: :api} do
+    Proxy.forward conn, path, "http://resource/minister-positions/"
+  end
+
+  match "/minister-position-functions/*path", %{ accept: [:json], layer: :api} do
+    Proxy.forward conn, path, "http://resource/minister-position-functions/"
+  end
+
+  match "/financering-codes/*path", %{ accept: [:json], layer: :api} do
```

Comment:
This sounds like the dutch `financiering` but misspelled :smile:. I guess this should be `financing-code` instead? Must have missed that in the previous PR since the domain.json file already includes that. sorry :smile:.

Reply:
That's good! Better late than never 😅